lib/common/reset-search-paths.ts

```diff
@@ -44,15 +44,18 @@ const makeElectronModule = (name: string) => {
 
 makeElectronModule('electron');
 makeElectronModule('electron/common');
+const electronModuleNames = new Set(['electron', 'electron/common']);
```

Comment:
Aren't all of these supposed to be exposed for all processes as @MarshallOfSound said in https://github.com/electron/electron/pull/35915#issuecomment-1269893433?

> It is intentional they work everywhere, they're superficial for TS only.

Reply:
This is a breaking change and as mentioned above the current behavior is intentional and actively in use.

This change serves no purpose other than to break existing code.